app/cdap/components/PostRunActions/WizardNav/WizardSelectActionStep/index.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Button, Tooltip, withStyles } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+export interface IWizardSelectActionStepProps {
+  postActionsList: {};
+  pluginFetch: (action) => void;
+  setShowValidateButton: (show: boolean) => void;
+}
+
+const ActionButton = styled.button`
```

Comment:
Try using mui `styled(Button)`

Reply:
The Mui Button has different styles and also capitalized letters. I kinda don't want to deal with it...